include/vast/Interfaces/AST/DeclInterface.td

```diff
@@ -92,11 +104,12 @@ def VAST_VarDeclInterface : VAST_ASTOpInterface< "VAST_VarDeclInterface",
         InterfaceMethod< "clang::VarDecl::hasGlobalStorage", "bool", "hasGlobalStorage", (ins) >,
         InterfaceMethod< "clang::VarDecl::isExceptionVariable", "bool", "isExceptionVariable", (ins) >,
         InterfaceMethod< "clang::VarDecl::isInitCapture", "bool", "isInitCapture", (ins) >,
-        InterfaceMethod< "clang::VarDecl::getInit", "void", "getInit", (ins) >
+        InterfaceMethod< "clang::VarDecl::getInit", "vast::ast::ExprInterface", "getInit", (ins) >,
+        InterfaceMethod< "clang::VarDecl::getASTContext", "vast::ast::ASTContextInterface", "getASTContext", (ins) >
```

Comment:
This it not part of `VarDecl` but it should be inhereted from `DeclInterface`.

Please double-check that you have methods on correct interfaces.

Reply:
Oh, you are right about the second one. The first one belongs to the `VarDeclInterface`.